pkg/minikube/cluster/cluster.go

```diff
@@ -165,6 +165,24 @@ func GetLocalkubeStatus(api libmachine.API) (string, error) {
 	}
 }
 
+// GetHostDriverIP gets the ip address of the current minikube cluster
+func GetHostDriverIP(api libmachine.API) (net.IP, error) {
```

Comment:
Nice, I can't believe we didn't have something like this before.  Out of scope for this PR, but we might want to consider moving some of our other functions that just call Driver.GetIP() directly to use this, so we can pass around a real `net.IP` struct.